src/index.js

```diff
@@ -442,19 +442,22 @@ export const order = style({
 export const gridGap = style({
   prop: 'gridGap',
   transformValue: px,
-  key: 'space'
+  key: 'space',
+  scale: defaultScale
```

Comment:
I don't think you want to use `defaultScale` here because this would always use the default scale even when a different scale is set in the theme, right?

Btw, thanks for the PR to fix this!

Reply:
Thanks for the PR!

That's right about the `scale` argument – though this is making me wonder whether or not this should be considered a breaking change. If someone is using `gridGap` with a value of, say, `1` then this would cause some layout issues

Reply:
I think this will be a breaking (but definitely desirable) change.

Reply:
Yeah, looks like it is breaking change even if it won't break any real code ;-)